src/extensions/default/CodeFolding/foldhelpers/handlebarsFold.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Fold range finder for handlebars/mustache template type files.
+ * @author Patrick Oladimeji
+ * @license MIT
+ * @date 14/08/2016 22:04:21
+ */
```

Comment:
I _think_ you have to use the standard license comment of adobe on newer file now the code folding extension is in the core.